p2p/net/swarm/swarm_metrics.go

```diff
@@ -77,6 +77,15 @@ var (
 		},
 		[]string{"outcome", "num_dials"},
 	)
+	dialLatency = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricNamespace,
+			Name:      "dial_latency_seconds",
+			Help:      "time taken to establish connection with the peer",
+			Buckets:   []float64{0.001, 1.3, 35},
```

Comment:
Where are these numbers coming from?

Reply:
I copied it from somewhere :sweat_smile: 
The range is 1ms to 10s